packages/cli/src/serve.js

```diff
@@ -78,9 +78,9 @@ function handleRequest(req, serveDirectory) {
   const pathname = parsedUrl.pathname || ''
 
   // get location of hyperparam assets
-  const hyperparamPath = import.meta.url
+  const hyperparamPath = decodeURIComponent(import.meta.url
```

Comment:
I think we should probably decode the url, and _then_ do the replacements? Maybe doesn't matter but currently if `file://` is uri-encoded, the following replacement will fail?

```suggestion
  const hyperparamPath = decodeURIComponent(import.meta.url)
```

Reply:
Will refactor and push code according to above !!